docs/pages/server-access/getting-started.mdx

```diff
@@ -277,21 +325,40 @@ Now, `tele-admin` has the ability to SSH into other Nodes within the cluster, tr
 
 ### Display the unified resource catalog
 
+<ScopedBlock scope={["oss", "enterprise"]}>
 Execute the following command within your bastion host console:
 
 ```code
 # List Nodes
-$ sudo tctl nodes ls
+$ tctl nodes ls
```

Comment:
I'm not sure what we mean by "unified resource catalog" in the title of this section, but `nodes ls` is only going to list SSH servers, so I'm not sure if I'd call that "unified resources."